db/migrate/20210615043807_validate_offers_and_offer_conditions_foreign_keys.rb

```diff
@@ -0,0 +1,6 @@
+class ValidateOffersAndOfferConditionsForeignKeys < ActiveRecord::Migration[6.1]
+  def change
+    validate_foreign_key 'offers', 'application_choices'
+    validate_foreign_key 'offer_conditions', 'offers'
```

Comment:
Do we need a data migration to clean up existing broken data? Or does this handle it?

Reply:
We couldn't delete any data so I don't think there is any corrupt data right now?

Reply:
Ah of course 👍